habitat/utils/visualizations/utils.py

```diff
@@ -192,7 +191,47 @@ def observations_to_image(observation: Dict, info: Dict) -> np.ndarray:
     assert (
         len(egocentric_view_l) > 0
     ), "Expected at least one visual sensor enabled."
-    egocentric_view = np.concatenate(egocentric_view_l, axis=1)
+
+    shapes = [x.shape for x in egocentric_view_l]
```

Comment:
Let code speak for itself:
```suggestion
    shapes_are_equal = np.all([x.shape for x in egocentric_view_l] == egocentric_view_l[0].shape)
```